astroquery/xmm_newton/core.py

```diff
@@ -0,0 +1,273 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+
+@author: Elena Colomo
+@contact: ecolomo@esa.int
```

Comment:
this needs to be a github contact. In fact it's very much preferred if development is happening with people using their own github account, so they can be tagged if neccessary when big reports are arriving.
For code credit and history, the git history and `git blame` command will provide all the sufficient information, too.